Content.Server/Speech/EntitySystems/BarkAccentSystem.cs

```diff
@@ -32,7 +39,10 @@ public string Accentuate(string message)
             }
 
             return message.Replace("!", _random.Pick(Barks))
-                .Replace("l", "r").Replace("L", "R");
+                .Replace("l", "r").Replace("L", "R")
+                //Corvax-Localization-Start
+                .Replace("л", "р").Replace("Л", "Р");
+                //Corvax-Localization-End
```

Comment:
Задевается изменением 2 строка
```suggestion
                //Corvax-Localization-Start
                .Replace("l", "r").Replace("L", "R")
                .Replace("л", "р").Replace("Л", "Р");
                //Corvax-Localization-End
```